release-notes/opensearch-release-notes-2.19.0.md

```diff
@@ -0,0 +1,1163 @@
+* Feat: only support visual editor alerts to navigate to discover([#368](https://github.com/opensearch-project/dashboards-assistant/pull/368))
+* Feat: return 404 instead of 500 for missing agent config name([#384](https://github.com/opensearch-project/dashboards-assistant/pull/384))
+* Feat: Update UI for In-context summarization in Alerts table([#392](https://github.com/opensearch-project/dashboards-assistant/pull/392))
+* Feat: Hide "stop generation" and regenerate button based on feature flag([#394](https://github.com/opensearch-project/dashboards-assistant/pull/394))
+* Feat: Chatbot entry UI redesign([#396](https://github.com/opensearch-project/dashboards-assistant/pull/396))
+* Feat: Set logo config in assistant and read logo by config([#401](https://github.com/opensearch-project/dashboards-assistant/pull/401))
+* Feat: Update dropdown list button label and remove popover title([#407](https://github.com/opensearch-project/dashboards-assistant/pull/407))
+* Feat: Use feature flag to disable the delete conversation api([#409](https://github.com/opensearch-project/dashboards-assistant/pull/409))
+* Feat: Disable the rename conversation api using feature flag([#410](https://github.com/opensearch-project/dashboards-assistant/pull/410))
+* Add query assistant summary to the assistant dropdown list ([#395](https://github.com/opensearch-project/dashboards-assistant/pull/395))
+
+
+### Opensearch Anomaly Detection
+
+
+* Allow triggering anomaly only on drop or rise of features ([#1358](https://github.com/opensearch-project/anomaly-detection/pull/1358))
+* Add flattens custom result index when enabled ([#1401](https://github.com/opensearch-project/anomaly-detection/pull/1401)), ([#1409](https://github.com/opensearch-project/anomaly-detection/pull/1409))
+
+
+### Opensearch Anomaly Detection Dashboards
+
+
+* Add feature direction and moving suppression rules to each feature ([#960](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/960))
+
+
+### Opensearch Flow Framework
+
+
+* Add multitenant remote metadata client ([#980](https://github.com/opensearch-project/flow-framework/pull/980))
+* Add synchronous execution option to workflow provisioning ([#990](https://github.com/opensearch-project/flow-framework/pull/990))
+
+
+### Opensearch Geospatial
+
+
+* Introduce new Java artifact geospatial-client to facilitate cross plugin communication. ([#700](https://github.com/opensearch-project/geospatial/pull/700))
+
+
+### Opensearch Neural Search
+
+
+* Pagination in Hybrid query ([#1048](https://github.com/opensearch-project/neural-search/pull/1048))
+* Implement Reciprocal Rank Fusion score normalization/combination technique in hybrid query ([#874](https://github.com/opensearch-project/neural-search/pull/874))
+
+
+### Opensearch Observability
+
+
+* Remove the maxFilesPerTrigger limits for VPC MV ([#2318](https://github.com/opensearch-project/dashboards-observability/pull/2318))
+
+
```

Comment:
Extra newlines?

Reply:
same format all across and previous releases. Doesn't really matter when rendered.